.github/pull_request_template.md

```diff
@@ -1,14 +1,32 @@
 <!--
 Thank you for your contribution to iTwinUI.
-Please describe your PR here and make sure to complete all of the items below before submitting.
+
+If you are only making changes to the docs site using the "Edit page on GitHub" link,
+then you can ignore most of this template.
+-->
+
+## Changes
+
+<!--
+What kind of code changes does this PR include?
+Mention anything that could be helpful for reviewers and include screenshots for visual changes.
+-->
+
+## Testing
+
+<!--
+How did you test your changes?
+If your PR has visual changes, then make sure they are demonstrated in html test pages as well as
+storybook, then approve visual test images for both (`yarn approve:css` and `yarn approve:react`).
+
+If not applicable, you can write "N/A".
 -->
 
-Closes # <!-- Add issue number -->
```

Comment:
Should we keep this? Or maybe add a comment about linking the issue? Because maybe external contributors might miss this step.

Reply:
I kept it initially but thought about it some more and removed it later in [`c33558a`](https://github.com/iTwin/iTwinUI/pull/977/commits/c33558a3053a0f1e76bd81fd6ccb50a355f5ffec) because:
- 99% of our PRs are from our own team members who will likely not miss step.
- there is not always an issue to link (e.g. quick fixes / docs changes)
- external contributors likely have an issue that they themselves would like to see closed, so they might link it themselves.
- even if it gets missed, it is super easy to link it through github's sidebar.